src/core/Expander/ExpanderTitleButton/ExpanderTitleButton.tsx

```diff
@@ -4,19 +4,21 @@ import classnames from 'classnames';
 import { withSuomifiDefaultProps } from '../../theme/utils';
 import { TokensProp, InternalTokensProp } from '../../theme';
 import { HtmlDiv, HtmlButton, HtmlButtonProps } from '../../../reset';
-import { expanderTitleButtonBaseStyles } from './ExpanderTitle.baseStyles';
+import { expanderTitleButtonBaseStyles } from './ExpanderTitleButton.baseStyles';
 import { Icon } from '../../Icon/Icon';
 import { ExpanderConsumer, ExpanderTitleBaseProps } from '../Expander/Expander';
 import { VisuallyHidden } from '../../../components';
 
 const baseClassName = 'fi-expander';
-const titleClassName = `${baseClassName}_title`;
+const titleClassName = `${baseClassName}_title-button`;
```

Comment:
Could the `titleClassName` to be the `baseClassName` instead. If the `baseClassName` is used to refer to the this component in this file.

Reply:
Probably should be, will test to ensure there are no side-effects.

Reply:
Fixed.